dockerfiles/build.include

```diff
@@ -70,8 +77,9 @@ init() {
       --dockerfile:*)
         DOCKERFILE="${1#*:}"
         shift ;;
-      --build-arg:*)
-        BUILD_ARG="--build-arg ${1#*:}"
+      --build-args:*)
```

Comment:
should we keep backward compliance with --build-arg ?

Reply:
yeah, I thought about this... but not sure how to do it except keep old arg name... any idea?

Reply:
I guess, users quite seldom use this argument by manually. In most cases I think they use original images...

Reply:
we are talking not only about users here, for instance after merge of this PR I will need to fix several CI jobs... but I do not see how to make it backward compatible without renaming OR duplication args

Reply:
@riuvshin maybe in your case it's handling more usecase than expected but I'm fine (like --build-argument:foo)